samcli/commands/local/start_api/cli.py

```diff
@@ -33,6 +33,14 @@
 and point SAM to the directory or file containing build artifacts.
 """
 
+COMMAND_SUGGESTIONS = """
```

Comment:
Can we make this a template that can be shared across command? Then, for each applicable command, we just provide a list of potential next commands. 
This will ensure consistency for this message across commands.